data/wordlists/tomcat_mgr_default_pass.txt

```diff
@@ -5,3 +5,9 @@ root
 tomcat
 s3cret
 vagrant
+QLogic66
+password
+Password1
+changethis
+r00t
+toor
```

Comment:
Seem to be missing the following entries here from tomcat_mgr_default_userpass.txt:

```
password1

j2deployer
OvW*busr1
kdsxc
owaspba
ADMIN
xampp
QLogic66
```

Reply:
Verified resolved in d1fb2a7bdef1e1246cd64d53f2daaa4259d4d9fb